webapp/src/components/formatPicker.tsx

```diff
@@ -36,28 +36,13 @@ const FormatPicker = (properties: FormatPickerProps) => {
     >
       <StyledDialogTitle>
         {t("num_fmt.title")}
-        <Cross onClick={handleClose} onKeyDown={() => {}}>
-          <svg
-            width="16"
-            height="16"
-            viewBox="0 0 16 16"
-            fill="none"
-            xmlns="http://www.w3.org/2000/svg"
-          >
-            <title>Close</title>
-            <path
-              d="M12 4.5L4 12.5"
-              stroke="#333333"
-              strokeLinecap="round"
-              strokeLinejoin="round"
-            />
-            <path
-              d="M4 4.5L12 12.5"
-              stroke="#333333"
-              strokeLinecap="round"
-              strokeLinejoin="round"
-            />
-          </svg>
+        <Cross
+          onClick={handleClose}
+          title="Close Dialog"
+          tabIndex={0}
+          onKeyDown={() => {}}
```

Comment:
* tabIndex and onKeyDown are not used here.
* Inside the widget we shouldn't use text as in "Close Dialog", you should add the string here:
https://github.com/ironcalc/IronCalc/blob/main/webapp/src/locale/en_us.json
And then use the translation tool. See how it is dine for the "save" button.

Reply:
I've removed it for now. I'll work on this next.